src/org/jitsi/impl/neomedia/transform/srtp/BaseSRTPCryptoContext.java

```diff
@@ -268,7 +268,7 @@ protected BaseSRTPCryptoContext(
      * @param pkt the RTP packet to be authenticated
      * @param rocIn Roll-Over-Counter
      */
-    synchronized protected void authenticatePacketHMAC(RawPacket pkt, int rocIn)
+    protected void authenticatePacketHMAC(RawPacket pkt, int rocIn)
```

Comment:
How is this related to the subject of this PR and/or OveruseEstimator? @champtar, could I please ask you to keep PRs to a single, standalone subject? If you need to work on two dependent subjects, you can still create two PRs and state in one of them that it depends on the other and paste a link to it.